baricadr/tasks.py

```diff
@@ -14,149 +14,97 @@
 
 
 app = create_app(config='../local.cfg', is_worker=True)
+app.app_context().push()
 celery = create_celery(app)
 
 
-# TODO [HI] test this
```

Comment:
Is it tested now? Should it?

Reply:
I manually tested it (by raising an exception). We don't really have an internal way of failing the task otherwise (that we could trigger for an automated test).
Unless we manually feed it a wrong task id in the wait_for field maybe?

Reply:
Yep it's what I feared, no good way to test... I don't know exactly what it will do with a wrong wait_for field (just setting wait_for_failed, or throwing an exception), it would be cool if you can add a test for it

Reply:
We should be able to raise an Exception by setting wait_for to [None]
(https://docs.celeryproject.org/en/stable/_modules/celery/result.html#AsyncResult)